addons/calendar/models/calendar_event.py

```diff
@@ -711,7 +715,7 @@ def create(self, vals_list):
         # is already given (coming from Google event for example).
         vals_list = [
             dict(vals, attendee_ids=self._attendees_values(vals['partner_ids']))
-            if 'partner_ids' in vals and 'attendee_ids' not in vals
+            if 'partner_ids' in vals and len(vals.get('attendee_ids', [])) == 0
```

Comment:
i'm not sure why attendee_ids would be in vals? for existing instance with the fix?

Reply:
If the event is imported from googleCal/Outlook, i believe they set attendee in vals (according to the comm just above)

Reply:
then in this case isn't it accurate? if google is saying there is no attendee, we should still not have attendees I think.

In any case the change above should only happen from calendar view and the google sync does not come from calendar view so it I'm not sure why this would be needed.